### DISCUSSION TITLE
raccolta di screenshot per creare URL di immagini da usare nel progetto

### DISCUSSION THREAD

**Discussion by cirospat** [opendatasicilia/ods-mkdocs-material#26] [Idee/Proposte]
# https://squidfunk.github.io/mkdocs-material/ homepage progetto mkdocs-material

![image](https://user-images.githubusercontent.com/3757525/138733754-cd859f4a-e97f-41a7-a752-d009c4fc6d02.png)

**Comment by aborruso** [comment]
non ho capito :(

**Comment by cirospat** [comment]
@aborruso 
faccio screenshot e li incollo qui in questo thread di issue.
Github mi genera l'URL dell'immagine, un URL pubblico, visibile da chiunque con il link.
Uso l'URL delle immagini per inserire immagini nelle pagine del progetto.
So che devo usare la cartella `img` per queste cose, .... tuttavia spesso la cartella `img` si trova dentro diverse directory e in passato mi sono confuso. 

E allora ho pensato: mi creo una issue dove metto le immagini e so che sono tutte quì, insomma è una cosa scrausa che mi sono inventato io .... fuori dagli schemi classici ...

**Comment by aborruso** [comment]
@cirospat ok, l'ho fatta diventare una discussione in "proposte/idee"
